src/Alert/Alert.stories.js

```diff
@@ -0,0 +1,177 @@
+// @flow
+import * as React from "react";
+import { storiesOf, setAddon } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import styles from "@sambego/storybook-styles";
+import chaptersAddon from "react-storybook-addon-chapters";
+import { withKnobs, text, boolean, select } from "@storybook/addon-knobs/react";
+
+import * as Icons from "../icons";
+
+import Alert from "./index";
+
+setAddon(chaptersAddon);
+
+const options = {
+  showSource: true,
+  allowSourceToggling: false,
+  showPropTables: false,
+  allowPropTablesToggling: false,
+};
+
+storiesOf("Alert", module)
+  .addDecorator(withKnobs)
+  .addDecorator(
+    styles({
+      padding: "20px",
+    }),
+  )
+  .addWithChapters("Default", () => {
+    const message = "The quick, brown fox jumps over a lazy dog.";
+    return {
+      info:
+        "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => <Alert>{message}</Alert>,
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Info alert", () => {
+    const title = text("Title", "");
+    const message = text("Message", "The quick, brown fox jumps over a lazy dog.");
+    const noIcon = boolean("No icon", false);
```

Comment:
Why ? I think is better to specify Icon instead of `noIcon` if Icon isn't specified just don't render it 💪

Reply:
Because when you don't fill the `Icon` prop, there should be rendered a default icon according to the type of Alert. If you will use `noIcon` prop, there will be Alert without the icon. Do you think I can solve it in a better way?

Reply:
just pass in icon prop `null`